src/helpers/utils.js

```diff
@@ -85,9 +85,10 @@ export const reactionHandler = async (message, currentServer, client) => {
   const words = loweredContent.split(" ");
   if (isAbcd(words)) await message.react(currentServer.eyeReactId);
 
-  if (Math.random() < 0.8) return; // Limit Ewibot react frequency
+  const frequency = Math.random() > 0.8 ? true : false; // Limit Ewibot react frequency
```

Comment:
Just replace it with "const frequency = Math.random() > 0.8 

It already returns true or false :)